user-guide/connecting.rst

```diff
@@ -6,6 +6,9 @@ Connecting to Isambard
 Linux
 .....
 
+Linux
+.....
+
```

Comment:
Does this just duplicate the heading?

Reply:
Yes that is spurious will fix and update.

Noted that connecting with current putty advice doesn't cover ssh keys so will raise separate issue for that.